sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationTransliterateOptions.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System.Collections.Generic;
+
+namespace Azure.AI.Translation.Text
+{
+    /// <summary> Client options for TextTranslationClient.Transliterate </summary>
+    public partial class TextTranslationTransliterateOptions
```

Comment:
nit: The name of this class is a little confusing. A user might ask: "Is this doing translation or transliteration?" I wonder if calling it "TransliterateOptions" makes more sense. Could be a good question to discuss during an architecture review.

Reply:
We will bring the name of this class up in our API review, to be scheduled this month.